### DISCUSSION TITLE
Asking For XAML Material Design 3 Elements

### DISCUSSION THREAD

**Discussion by melgoharyme** [unoplatform/uno#19053] [Q&A] [answered]
How can I get XAML Material Design 3 elements without using plugins like Figma to XAML?

**Comment by morning4coffe-dev** [comment]
Hey @melgoharyme,  

You can use `Uno.Material` with any application. Check out the [getting started docs](https://platform.uno/docs/articles/external/uno.themes/doc/material-getting-started.html), or when creating a new project, you can select the Material theme in the creation wizard:  
![image](https://github.com/user-attachments/assets/a5819b5e-72ee-4361-831d-8ed557992e6a)  

Let me know if this helps!

**Reply by melgoharyme** [reply]
Actually I do use it, I would like to request XAML elements like buttons, data fields, checkpoints, etc.

**Reply by morning4coffe-dev** [reply, marked as answer]
You don't see Material design on Buttons, TextBoxes etc.? Can you provide a repro, please? You should see all the elements styled with the same design as you can see in https://gallery.platform.uno/ for example.
